truffle/src/main/java/org/jruby/truffle/core/encoding/EncodingNodes.java

```diff
@@ -483,6 +484,16 @@ public DynamicObject getEncoding(int index) {
 
     }
 
+    @Primitive(name = "encoding_minlength", needsSelf = false)
+    public static abstract class EncodingMinLengthPrimitiveNode extends PrimitiveArrayArgumentsNode {
+
+        @Specialization
+        public int minLength(DynamicObject encoding) {
+            return EncodingUtils.encMbminlen(Layouts.ENCODING.getEncoding(encoding));
```

Comment:
I'm not sure the utility method adds much here. I guess it looks more like MRI's code, but it seems to be a bit of a useless wrapper.